tests/Test.ADAL.NET.Unit/JsonWebTokenTests.cs

```diff
@@ -122,11 +122,21 @@ public void Initialize()
             AuthenticationResult result = await context.AcquireTokenAsync(TestConstants.DefaultResource, clientAssertion, true).ConfigureAwait(false);
             Assert.IsNotNull(result.AccessToken);
 
+            HttpMessageHandlerFactory.AddMockHandler(X5CMockHandler);
```

Comment:
>HttpMessageHandlerFactory.AddMockHandler(X5CMockHandler); [](start = 12, length = 57)

is this needed? you already do this above

Reply:
yes, we have a que of mocked internet responses and each time a response is returned to the test it is removed from the que